pkg/asset/ignition/recovery_ignition.go

```diff
@@ -6,14 +6,25 @@ import (
 
 	configv32 "github.com/coreos/ignition/v2/config/v3_2"
 	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/go-openapi/swag"
 	"github.com/openshift/appliance/pkg/asset/config"
 	"github.com/openshift/appliance/pkg/asset/manifests"
 	"github.com/openshift/appliance/pkg/installer"
 	"github.com/openshift/installer/pkg/asset"
+	ignasset "github.com/openshift/installer/pkg/asset/ignition"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// InteractiveUIFilePath is the sentinel file path that indicates interactive UI should be enabled
+	InteractiveUIFilePath = "/etc/assisted/interactive-ui"
```

Comment:
@andfasano Does appliance also need to embed `/etc/assisted/extra-manifests/internalreleaseimage.yaml` file? Is there a sample?